semanticdb/integration/src/main/scala/example/Types.scala

```diff
@@ -43,6 +43,7 @@ object Test {
 
     val singleType1: x.type = ???
     val singleType2: p.x.type = ???
+    val Either = scala.util.Either
```

Comment:
For future reference, we should put test cases like this in a separate `Bugs.scala` file (name up for debate) where we only append new code. Adding a new line to Types.scala shifts all positions in the following statements which causes unnecessary diffs making it harder to review the changes.

Reply:
I agree: it's also probably interesting to keep track of this bugs (and respective workaround) in case they get fixed upstream